include/verilated_types.h

```diff
@@ -124,11 +124,17 @@ class VlProcess final {
     void state(int s) { m_state = s; }
     void disable() {
         state(KILLED);
-        disable_fork();
+        disableFork();
     }
-    void disable_fork() {
+    void disableFork() {
         for (VlProcess* childp : m_children) childp->disable();
     }
+    bool completed() { return state() == FINISHED || state() == KILLED; }
+    bool completedFork() {
+        for (VlProcess* childp : m_children)
+            if (!childp->completed()) return false;
+        return true;
+    }
```

Comment:
and fix state() to be const.

```suggestion
    bool completed() const { return state() == FINISHED || state() == KILLED; }
    bool completedFork() const {
        for (const VlProcess* const childp : m_children)
            if (!childp->completed()) return false;
        return true;
    }
```